common/placer_heap.cc

```diff
@@ -401,6 +401,14 @@ class HeAPPlacer
         // Initial constraints placer
         for (auto &cell_entry : ctx->cells) {
             CellInfo *cell = cell_entry.second.get();
+
+            if (cell->bel != BelId()) {
+                if (ctx->verbose)
+                    log_error("Cell \'%s\' (%s) has already been constrianed by arch implementation",
```

Comment:
an error should not be gated by `ctx->verbose` (also, s/constrianed/constrained)